.github/workflows/e2e.yaml

```diff
@@ -2,6 +2,8 @@ name: E2E Tests
 
 on:
   pull_request:
+    branches:
+      - does-not-exist
```

Comment:
and I guess after moving the content of `deleteme` file to this file, we should remove this line